CHANGELOG

```diff
@@ -1,5 +1,174 @@
+* Add validation for state id mappings. (#6680)
+* Fix #6043: Change state's content_ids_to_audio_translations to recorded_voiceovers (#6587)
+* Fix #6548: Convert the current per-exploration translator role to a voice-artist role. (#6566)
+
+Testing coverage
+* Fix backend tests on develop (#6878)
+* Fix backend tests on develop (#6875)
+* Fix part of #6550: Write tests for domain/topic_domain (#6873)
+* Fix part of #6550: Write tests for domain/question_services (#6854)
+* Fix part of #6550: Write tests for controllers/editor (#6849)
+* Fix part of #6550: Write tests for domain/user_jobs_one_off (#6842)
+* Fix part of #6550: Write tests for domain/collection_services (#6839)
+* Fix part of #6550: Write tests for domain/classifier_domain and domain/event_services (#6831)
+* Fix part of #4057: Add unit tests for lookupEnvs method of ExpressionSyntaxTreeService.ts (#6828)
+* Fix part of #6550: Write tests for controllers/suggestion (#6822)
+* Fix part of #6550: Write tests for collection_domain (#6819)
+* Fix part of #6550: Make coverage 100% for controllers/library (#6812)
+* Fix part of #6550: Write tests for storage base_model, statistics and pages.py (#6811)
+* Fix part of #6550: Write tests for classifier, collection, email, feedback and user storage gae_models (#6802)
+* Fix part of #6550: Write tests for gae_image_services and replaced PIL with Pillow (#6788)
+* Fix part of #6550:  Write tests for controllers/acl_decorators (#6782)
+* Fix part of #6550: Write tests for controllers/collection_editor (#6764)
+* Fix part of #6550: Improve test coverage of core/controller/libary.py" to 92% (#6760)
+* Fix part of #6550: Write tests for controllers/email_dashboard (#6744)
+* Fix part of #6550: Write tests for collection_viewer and translator (#6741)
+* Fix part of #6550: Improve coverage for learner dashboard (#6729)
+* Fix #6572: Adds e2e test for written translation project (#6718)
+* Fix part of #6550: Write tests for core/controllers/profile.py  (#6713)
+* Fix part of #6550: Improved coverage for controllers/base.py (#6712)
+* Fix part of #6550: Write backend tests for skill_jobs_one_off, story_jobs_one_off,  user_query_jobs_one_off , value_generators_domain, and visualization_registry (#6710)
+* Fix part of #6550: Write backend tests for core/domain/param_domain.py and core/domain/search_services.py (#6664)
+* Fix part of #6550: Increased coverage for learner progress services (#6643)
+* Fix part of #6550: Write backend tests for core.controllers.moderator and core.controllers.question_editor_test (#6585)
+* Fix part of #5134: Improve coverage for creator_dashboard and gae_search_services (#6480)
+* Fix part of #6550: Adds 100% backend tests coverage for resources.py, topics_and_skills_dashboard.py, activity_jobs_one_off.py and classifier_services.py (#6602)
+
+Improvements to editors and players
+* Fix #6262: Constrain the width of the tooltip so that it fits the screen (instead of overflowing)  (#6865)
+
```

Comment:
Minor nit: add one line below this one (just to follow the existing file format).

Reply:
Oops. My bad. I thought I scrolled below and saw just one newline. Thanks for pointing it out!